Sources/CircularProgress/CircularProgress.swift

```diff
@@ -74,6 +79,12 @@ public final class CircularProgress: NSView {
 				self.progressCircle.progress = self._progress
 			})
 
+			if progress == 0 && isIndeterminate {
+				progressLabel.isHidden = cancelButton.isHidden
+			} else {
+				progressLabel.isHidden = !cancelButton.isHidden
+			}
```

Comment:
I would prefer a ternary here.

Reply:
I chose against that because it's not the simplest `if`-statement, but if you're ok with it. 👍